packages/samples/angular/src/app/atomic-angular-page/atomic-angular-page.component.html

```diff
@@ -1,6 +1,51 @@
 <atomic-search-interface searchHub="MainSearch" pipeline="Search">
   <div class="search">
-    <atomic-search-box></atomic-search-box>
+    <atomic-search-box>
+      <atomic-search-box-recent-queries></atomic-search-box-recent-queries>
+      <atomic-search-box-instant-results>
+        <atomic-result-template>
+          <template>
+            <style>
```

Comment:
Yeah the Angular rendering engine preprocess style tag and just removes them from the .HTML file. 

It expects all styling to be in the .css files.

IIRC the way to do it is to wrap components, and then style the wrapper, instead of trying to style the original atomic component.


ie: instead of having this in your HTML template:
```
<div class="rating-wrapper">
  <atomic-result-rating field="ec_rating"></atomic-result-rating>
  <atomic-field-condition class="field" if-defined="ec_rating">
    <span>
      <atomic-result-number field="cat_rating_count"></atomic-result-number>
    </span>
  </atomic-field-condition>
</div>
```
Which you try to style directly, you create a custom `<styled-rating>` component in the Angular project, which can then itself contains the style you need, and takes care of outputting the Atomic component.


The other way, if I remember correctly, is to deal with the CSS as a string in the `.ts` file, and inject it in the template HTML element.

Reply:
aah cool! thanks for clarifying, I'm going to skip that step here 😬  but good to know!!